org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/managers/GradleBuildSupport.java

```diff
@@ -108,4 +109,12 @@ public boolean fileChanged(IResource resource, CHANGE_TYPE changeType, IProgress
 	public static void saveModels() {
 		CorePlugin.listenerRegistry().dispatch(new WorkbenchShutdownEvent());
 	}
+
+	@Override
+	public void updateBuildFile(IProject project) throws CoreException {
+		IFile file = project.getFile(GradleProjectImporter.BUILD_GRADLE_DESCRIPTOR);
+		if (file.exists()) {
+			file.setLocalTimeStamp(System.currentTimeMillis() + 1000);
```

Comment:
why?

Reply:
It will cause updating of a gradle project. A user can disable it with the java.configuration.updateBuildConfiguration property.